src/main/java/cubicchunks/worldgen/dependency/DependencyManager.java

```diff
@@ -36,139 +36,92 @@
 import java.util.Map;
 import java.util.Set;
 
+/*
+ * TODO: Abstraction to allow for non-cube dependents.
+ * TODO: Commenting
```

Comment:
Just a note as you may not know it, when you fix it, add `Fixes #14` in your commit message to automatically close the issue when it's merged.